.gitignore

```diff
@@ -21,5 +21,7 @@ bin/
 *.suo
 *.log
 *.tlog
+*.bbl
+*.blg
```

Comment:
Please move this into dedicated `.gitignore` + add `*.sty` / `mtheme.sty` pattern:
- modules/gapi/doc/slides/.gitignore

Reply:
done